src/CUITe/ScreenObjects/Screen.cs

```diff
@@ -46,5 +46,26 @@ public static T Launch<T>(ProcessStartInfo applicationStartInfo) where T : Scree
                 SearchLimitContainer = application
             };
         }
+
+
+        /// <summary>
+        /// Creates a reference to an application from an existing process
```

Comment:
Change to `Creates a reference to an application from an existing process and returns a screen of type <typeparamref name="T"/>.`